### DISCUSSION TITLE
Condition v-if dans champs répétables

### DISCUSSION THREAD

**Discussion by jodionrq** [MTESSDev/FRW#34] [Annonces]
Bonjour,
Merci pour l'ajout de la méthode dans le P700U. 
Est-ce qu'il y a une autre information à ajouter dans la condition v-if lorsque celle-ci est dans un groupe répétable? 
Dans l'exemple il est mentionné : val(nomGroupe.nomChamp, occuranceDansGroupeSiGroupeRepetable).
Qu'est-ce qu'on doit ajouter comme "occurance"?
Pour l'instant, si je remplis mes champs dans mon premier groupe répétable, les affichages conditionnels apparaissent dans le 2e groupe sans que les cases ou boutons soient cochés.
Merci!

**Comment by j0ber** [comment]
Bonjour,
Pouvez-vous mettre le bout de code provenant de votre fichier de config s.v.p. ? avec aussi la ou les champs utilisés dans le v-if
Je vais regarder ça.

Aussi une impression d'écran et une indication du résultat attendu... Je ne comprend pas bien votre question.
Merci!

**Comment by j0ber** [comment]
Si le v-if utilise un champ qui est dans le même groupe et même occurence vous pouvez utiliser :

``v-if: val(`${name}.NomChamp`, index) === 'true'``

${name} va prendre comme valeur le nom du groupe
index va prendre comme valeur l'index de l'occurence courante

**Comment by jodionrq** [comment]
Bonjour,
Voici le code pour le groupe :
![code_groupe](https://user-images.githubusercontent.com/124822406/226730081-96433ac9-cce0-488b-a542-8a1b384a7d1d.png)

Voici le code pour le branching :
![code_condition](https://user-images.githubusercontent.com/124822406/226730209-34250539-4116-44bf-8133-cc752a02b75f.png)

Voici une capture d'écran dans le bac à sable. Dans le 1er groupe, les boutons radio doivent apparaitre lorsqu'une des cases "situation salariale" est cochée. Dans le 2e groupe, les boutons radio apparaissent même si aucune case n'est coché.
![image_FRW](https://user-images.githubusercontent.com/124822406/226730328-d035ebca-6fec-471a-91c8-22e65fac0509.png)

Si vous avez besoin de plus d'exemple, n'hésitez pas.
Merci!

**Comment by j0ber** [comment]
Bonjour,
J'ai mis à jour les exemples de v-if. Il y a maintenant un exemple dans un groupe répétable.
[Voir l'exemple de v-if dans un groupe répétable](https://formulaires.it.mtess.gouv.qc.ca/Form/7/P700U/0/N/#p=sectionConditionSi)

**Reply by jodionrq** [reply]
Bonjour, merci beaucoup pour l'exemple et votre extraordinaire collaboration, ç'a réglé mon problème!
Bonne journée
